python/tank/platform/qt/tankqdialog.py

```diff
@@ -299,44 +296,43 @@ def _format_context_property(p, show_type=False):
                 if p is None:
                     formatted = "Undefined"
                 elif show_type:
-                    formatted = "%s %s" % (six.ensure_str(p.get("type")),
-                                           six.ensure_str(p.get("name")))
+                    formatted = "{} {}".format(p.get("type"), p.get("name"))
                 else:
-                    formatted = "%s" % six.ensure_str(p.get("name"))
+                    formatted = "{}".format(p.get("name"))
 
                 return formatted
 
             tooltip = ""
             tooltip += "<b>Your Current Context</b>"
             tooltip += "<hr>"
-            tooltip += "<b>Project</b>: %s<br>" % _format_context_property(
-                self._bundle.context.project
+            tooltip += "<b>Project</b>: {}<br>".format(
+                _format_context_property(self._bundle.context.project)
```

Comment:
Why all these changes? Benefit vs risk?

Reply:
On second thought, I'll limit the change only to the affected function instead of the whole thing. Thanks for raising this.

Reply:
OK thanks. I have nothing against cleaning but I would prefer in a dedicated PR. Especially something like this one, fixing a regression, my preference is for the smallest change.